projects/gloo/api/v1/options/tracing/tracing.proto

```diff
@@ -43,6 +43,13 @@ message ListenerTracingSettings {
     repeated TracingTagEnvironmentVariable environment_variables_for_tags = 6;
     // Optional. If specified, Envoy will include the literals with the given tag as tracing tags.
     repeated TracingTagLiteral literals_for_tags = 7;
+    // Optional. If specified, Envoy will include tags from the dynamic metadata.
+    repeated TracingTagMetadata metadata_for_tags = 11;
+
+    // Optional
+    // Create separate tracing span for each upstream request if true. And if this flag is set to true, the tracing provider will assume that Envoy 
+    // will be independent hop in the trace chain and may set span type to client or server based on this flag. 
+    bool spawn_upstream_span = 10;
```

Comment:
There was something about T-mobile and their span name requirement on the request. I don't really know the detail but @ashishb-solo will you have some insight for span name if we are adding open tracing for AI?

Reply:
This is unrelated, that's about the parent span, this is the span for the upstream call